bridge/_files.php

```diff
@@ -20,7 +20,6 @@
     __DIR__ . '/../src/DDTrace/StartSpanOptionsFactory.php',
     __DIR__ . '/../src/DDTrace/Time.php',
     __DIR__ . '/../src/DDTrace/Transport/Http.php',
-    __DIR__ . '/../src/DDTrace/Transport/Stream.php',
```

Comment:
We'll also need to remove `Json.php` from the `$autoloaderMapping` array in `bridge/dd_register_optional_deps_autoloader.php`.

Reply:
Good catch @SammyK! For some reason it had remained un-pushed! https://github.com/DataDog/dd-trace-php/pull/1235/commits/d0e0706dda121fb84601462302f0cde8a5e5322e